src/BaseTemplates/ConsoleApp/project.json

```diff
@@ -5,12 +5,13 @@
   },
 
   "dependencies": {
-    "NETStandard.Library": "1.0.0-rc2-23830"
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc2-23930"
+    }
   },
 
   "frameworks": {
-    "netstandardapp1.5": {
-      "imports": "dnxcore50"
-    }
+    "netcoreapp1.0": {}
```

Comment:
We can't do this line yet until the rid inference change has gotten into the tooling. This doesn't work as far as I know. You can leave it as netstandardapp1.5 with no imports for now.